documentation/docs/getting_started/rust.mdx

```diff
@@ -52,78 +58,43 @@ brew install openssl@1.1
 OPENSSL_ROOT_DIR=$(brew --prefix openssl@1.1)
 ```
 
-### Linux
+</TabItem>
+<TabItem value={'linux'} label={'Linux'}>
 
 You can install _cmake_ and _openssl_ with your distro's package manager or download from their websites. On Debian and
 Ubuntu you will also need the [_build-essential_](https://packages.debian.org/sid/build-essential) package.
 
-## Include the Library in Your `Cargo.toml`
+
+</TabItem>
+</Tabs>
+
+## Include the Library
 
 The only thing you need to do to start using the library is to add it as dependency in your `Cargo.toml` file:
 
 ```toml {2}
 [dependencies]
 iota-wallet = { git = "https://github.com/iotaledger/wallet.rs", branch = "develop" }
+```
+
+### Enable Asynchronous Functionality
+
+The library uses asynchronous functionality. You can enable it by adding the following line as a dependency:
```

Comment:
I think this is a bit misleading. The library is asynchronous, there is nothing to enable. Maybe you mean that the user will need a runtime like tokio to execute the async parts, ant that's correct, but it's not optional or to be "enabled".

Reply:
Understood.  Thanks for the clarification! :)   Since it isn't optional as much as required I'll revert this change.